src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -122,10 +121,11 @@ def _init_data_path(container_id: "UniqueId", container_type: "ContainerType") -
 
     @ensure_disk_not_overutilize
     def enqueue_operation(self, op: Operation, *, wait: bool) -> None:
-        self._last_version = self._queue.put(op)
+        if not self._accepts_operations:
+            warn_once("Not accepting operations", exception=NeptuneWarning)
```

Comment:
@normandy7 IMO that's the only thing for you here. This will be triggered in case that callback with `run.stop()` inside was called (no progress/lag callback) and at the same time someone is still trying to log more metadata. It's quite general but I don't know any more cases that this could happen but probably they exist.

Reply:
I was asked to force this PR further. Let me know after all if you have any suggestions here.

Reply:
@Raalsky You mean there could be cases that are unrelated to the lag/no-progress callback? But the general scenario is that the run object is available in the session, it's just no longer active?